tests/configurations/mockPrisma.ts

```diff
@@ -0,0 +1,57 @@
+import { vi, type Mock } from 'vitest';
+import { mockDeep, type DeepMockProxy } from 'vitest-mock-extended';
+import type { PrismaClient } from '@prismaClient';
+
+/**
+ * Creates a deep mock Prisma client using vitest-mock-extended.
+ * This is required for Vitest 4.0+ which validates vi.spyOn() targets are functions.
+ * Prisma uses Proxy objects where methods don't exist until accessed.
+ *
+ * vitest-mock-extended handles all the complexity of mocking Prisma's nested structure.
+ *
+ * @returns A deep mock of PrismaClient with all methods mocked
+ */
+export function createMockPrisma(): DeepMockProxy<PrismaClient> {
```

Comment:
im not sure you actually need a reuseable fucntion for a one liner

Reply:
done